Packs/FeedGCPWhitelist/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### New: Google IP Ranges Feed
+Google IP Ranges Feed integration to get GCP and Google global IP ranges. This integration should be used instead of GCP Whitelist Feed.
```

Comment:
```suggestion
Use the *Google IP Ranges Feed* integration to get GCP and Google global IP ranges. This integration should be used instead of the GCP Whitelist Feed integration.
```